templates/styles/modal.css

```
@@ -72,17 +73,46 @@ body.modal-open {
+  color: white;
+}
 
 .modal-description > *:not(:last-child) {
   margin-bottom: 8px;
 }
 
 .modal-description h2 {
+  width: 80%;
+  word-break: keep-all;
+
   font-size: 2rem;
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)